packages/lib/src/components/Card/components/CardInput/CardInput.tsx

```diff
@@ -52,8 +52,10 @@ function CardInput(props: CardInputProps) {
     const [dualBrandSelectElements, setDualBrandSelectElements] = useState([]);
     const [selectedBrandValue, setSelectedBrandValue] = useState('');
 
+    const showBillingAddress = props.billingAddressMode !== 'none' && props.billingAddressRequired;
+
     const [storePaymentMethod, setStorePaymentMethod] = useState(false);
-    const [billingAddress, setBillingAddress] = useState(props.billingAddressRequired ? props.data.billingAddress : null);
+    const [billingAddress, setBillingAddress] = useState(showBillingAddress ? props.data.billingAddress : null);
```

Comment:
Seems to be still used on line 55. Do you think that qualifies as a breaking change @pabloai ?

Reply:
Yes, it would be a bad idea to not accept `billingAddressRequired` as it is what other components are using (Google Pay, Boleto, etc.) and it's also aligned with other prop names as `holderNameRequired `, `emailRequired`, `koreanAuthenticationRequired`, etc.

Reply:
@pabloai it seems `billingAddressRequired` is still used (in combination with the new `billingAddressMode`. Would you still consider it a breaking change?

Reply:
@marcperez Ah I see, that's correct. Then I don't see any issue with this.